packages/mrc-ide-covidsim/src/imperial-params.ts

```diff
@@ -132,6 +142,14 @@ export function assignParameters(
   // School closure
   p['Place closure start time'] = 0
   p['Duration of place closure'] = 10000
+
+  /*
+    If there are changes times / intervention periods a < b < c < d, then closure durations have
+    upper bounds of (b-a), (c-b), and (d-a). So yes you're right the value of 10000 is set so as to
```

Comment:
```upper bounds of (b-a), (c-b), and (d-a).```

Is this supposed to be:

```upper bounds of (b-a), (c-b), and (d-c).```

because, if not I don't understand.